src/AI/AIUtils.java

```diff
@@ -249,6 +250,23 @@ public static CommanderAbility queueCromulentAbility(Queue<GameAction> q, Comman
     return shootableTiles;
   }
 
+  /**
+   * @return The range at which the CO in question might be able to attack after moving.
+   */
+  public static int findMaxStrikeWeaponRange(Commander co)
```

Comment:
The only case I can think of where this wouldn't return 1 is with your proposed Bship rework... which we don't have.

Reply:
Hey man, I like being prepared.
I suppose I could hardcode in `return 1` for now, but I want the function to exist for possible future shenanigans.